modules/cnv-installing-qemu-guest-agent-on-windows-vm.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-installing-qemu-guest-agent.adoc
+
+[id="cnv-installing-qemu-guest-agent-on-windows-vm_{context}"]
+= Installing QEMU guest agent on a Windows virtual machine
```

Comment:
Since this isn't a procedure, don't use a gerund in the module title.

I'd move the contents of the module to the assembly.

Reply:
I'm going to create a Jira task for next sprint to revisit this as this will require removing this file if I do move the contents to the assembly. I agree this is the right thing to do, though. Created https://jira.coreos.com/browse/CNV-3586